src/pre_defined_classes/simple_programs_taking_string_user_input/hard/q10/PasswordWeakness.java

```diff
@@ -0,0 +1,45 @@
+package pre_defined_classes.simple_programs_taking_string_user_input.hard.q10;
+
+/*
+Passwords are one of the most important things that everyone in the digital world has to take care of. Some of the most
+popular passwords that are easily guessed by hackers contain the word "password" or the numbers "1234". Make a program
+that asks the user to enter a password, then checks if the password is strong or not. For this question, a password is
+weak if it can be easily guessed based on the criteria above (this means it contains the two parameters above).
+The program will print true or false depending on whether the inputted password is weak. Assume users always enter a
+password with at least 4 characters.
```

Comment:
Does it have to be 4 characters?

Reply:
Like if they just entered 'P', would the question not work?

Reply:
Yes, entering 'P' would work. I forgot why I added that in.